src/frontend/tsx/Challenges.tsx

```diff
@@ -89,22 +89,22 @@ export function Challenges () {
 
             return (
               <li className={`game challenge ${colorClass}`} key={key}>
-                <div className='row' style={{ justifyContent: 'space-between' }}>
+                <div className='challenge-box'>
                   <div className='row'>
                     <img
                       src={`https://raw.githubusercontent.com/lichess-org/lila/5a9672eacb870d4d012ae09d95aa4a7fdd5c8dbf/public/piece/cburnett/${mySide}N.svg`}
-                      height={60}
-                      width={60}/>
+                    />
                     <div className='col'>
-                      <p style={{ fontSize: '1.25rem' }}>{challenger}</p>
+                      <p className='challenger-name'>{challenger}</p>
                       <p
                         title={description}
-                        style={{ maxHeight: '5rem', overflow: 'hidden', textOverflow: 'ellipsis' }}>
+                        className='challenger-desc'
+                      >
                         {description}
                       </p>
                     </div>
                   </div>
-                  <div className='col' style={{ justifyContent: 'space-evenly', margin: '0.5em' }}>
+                  <div className='col'>
```

Comment:
Challenges still look correct even with this gone?

Reply:
Confirmed.